features/manage-position/Create.tsx

```diff
@@ -143,6 +145,7 @@ const Create = () => {
           const tokensWei = toWei(tokens);
           const tx = await emp.create([collateralWei], [tokensWei]);
           setHash(tx.hash as string);
+          notify?.hash(tx.hash);
```

Comment:
Should we remove the the transaction hash/success/error component from this and other files now?

Reply:
I thought about that. I dont really see any reason to leaving them both for now. Can strip them out if need be but it's useful to get both I think

Reply:
Ok we can strip out in another PR this one is chunky already

Reply:
It covers all functions already though. I'm happy to pull it into two PRs so we can test. 

Another option to "test in production" would be to circulate this testing URL internally for anyone creating a transaction and validate that no errors are thrown.

Reply:
I have removed notify.js to put into a separate PR. Cleaner this way.